pkg/cmd/template/schema_inspect_test.go

```diff
@@ -624,6 +623,84 @@ components:
 	})
 }
 
+func TestSchemaInspect_annotation_adds_title_key(t *testing.T) {
```

Comment:
Nice job on these tests. For organization, let's move the subtests from ` TestSchemaInspect_annotation_adds_title_key` to the test above: `TestSchemaInspect_annotation_adds_key`. That test tests adding 'documenting' keys when an annotation that provides documentation information is present. As opposed to the other schema annotations we have (`@schema/default`, `@schema/type`, `@schema/nullable`) add constraints to the schema.